packages/core/stories/overlay/overlay.stories.tsx

```diff
@@ -337,3 +337,28 @@ export const WithActions = ({ onOpenChange }: OverlayProps) => {
     </Overlay>
   );
 };
+
+export const WithTooltip: StoryFn<OverlayProps> = ({ ...args }) => {
```

Comment:
@pseys is it an issue for the user that overlay and tooltip look somewhat identical?
have a look at this example in Storybook, to verify

Reply:
Perhaps, once an overlay is displayed, then you do not need the tooltip anymore ?
Seems strange (perhaps) to have both ?

Reply:
> Perhaps, once an overlay is displayed, then you do not need the tooltip anymore ? Seems strange (perhaps) to have both ?

I've got an issue to solve that